sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroidOptions.java

```diff
@@ -7,6 +7,7 @@
 import io.sentry.SpanStatus;
 import io.sentry.protocol.SdkVersion;
 import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
 /** Sentry SDK options for Android */
 public final class SentryAndroidOptions extends SentryOptions {
```

Comment:
Yes, we decided to put some options only on `SentryAndroidOptions` and in case will ever add profiling to jvm backend, we'd "move" those options to `SentryOptions`, but it could take a while, as we don't have a current plan on that

Reply:
ah sorry I missed the comment thread about it, makes sense!